src/liquid.ts

```diff
@@ -24,6 +23,11 @@ const engine = Liquid ? new Liquid() : undefined
 export function fromLiquidTemplate<State extends object = DefaultState>(
     template: string
 ): (container: HTMLElement, state: State) => PromiseLike<void> {
+    const LiquidInstance =
```

Comment:
can this assignment be simplified?
e.g.
const LiquidInstance = window.liquidjs?.Liquid

also the const name is misleading. `engine` is the instance, `LiquidInstance` is the factory